tests/acceptance/features/bootstrap/TagsContext.php

```diff
@@ -1463,7 +1468,11 @@ public function sharedByHasNoTags($fileName, $sharingUser) {
 		// The array of tags has a single "empty" item at the start.
 		// If there are no tags, then the array should have just this
 		// one entry.
-		Assert::assertCount(1, $tagList);
+		Assert::assertCount(
+			1,
+			$tagList,
+			"Expected no tags, but got '" . \count($tagList) . "' tags"
```

Comment:
```suggestion
			"Expected no tags, but got '" . (\count($tagList) - 1) . "' tags"
```
Or maybe refactor the whole section. Something like:
```
$numTags = \count($tagList) - 1;
Assert::assertEquals(
  0,
  $numTags,
  "Expected no tags for " . $fileName . " but got " . $numTags . " tags"
);
```

Reply:
fixed